tests/tracer/test_memory_leak.py

```diff
@@ -64,28 +121,18 @@ def test_fork_open_span(tracer):
 
     if pid == 0:
         assert len(wd) == 1
+        del span
         gc.collect()
-        # span is still open and in the context
+        # span is still open in the context
         assert len(wd) == 1
         span2 = trace(wd, tracer, "span2")
         assert span2._parent is None
         assert len(wd) == 2
         span2.finish()
 
         del span2
-
-        # Expect there to be one span left (the original from before the fork)
-        # which is inherited into the child process but will never be closed.
-        # The important thing in this test case is all new spans created in the
-        # child will be gc'd.
```

Comment:
do we want to keep these comments?

Reply:
Oh strange - I mixed up the merge conflict resolution! I'll revert this